app/components/candidate_interface/reopen_banner_component.rb

```diff
@@ -0,0 +1,39 @@
+class CandidateInterface::ReopenBannerComponent < ViewComponent::Base
+  attr_accessor :phase, :flash_empty
+
+  def initialize(phase:, flash_empty:)
+    @phase = phase
+    @flash_empty = flash_empty
+  end
+
+  def show?
+    flash_empty &&
+      (show_apply_1_reopen_banner? || show_apply_2_reopen_banner?)
+  end
+
+private
+
+  def show_apply_1_reopen_banner?
+    apply_1? &&
+      EndOfCycleTimetable.show_apply_1_reopen_banner?
+    # FeatureFlag.active?(:deadline_notices)
```

Comment:
What's this comment for?

Reply:
Oops that's a mistake, we were debating whether we needed a feature flag or not but opted against

Reply:
Thanks for fixing everything Steve :)